file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs

```diff
@@ -1,24 +1,131 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
 using UKHO.FileShareService.DesktopClient.Core.Jobs;
+using UKHO.FileShareAdminClient;
+using UKHO.FileShareAdminClient.Models;
+using System;
+using System.Linq;
+using System.Net;
+using Newtonsoft.Json;
+using Microsoft.Extensions.Logging;
+using Prism.Commands;
+using System.Threading;
+using UKHO.FileShareService.DesktopClient.Core.Models;
 
 namespace UKHO.FileShareService.DesktopClient.Modules.Admin.JobViewModels
 {
     public class AppendAclJobViewModel : BaseBatchJobViewModel
     {
         private readonly AppendAclJob job;
+        private readonly Func<IFileShareApiAdminClient> fileShareClientFactory;
+        private string executionResult = string.Empty;
+        string responseMessage = string.Empty;
+        private bool isCommitting;
+        private bool isExecutingComplete;
+        private readonly ILogger<AppendAclJobViewModel> logger;       
 
-        public AppendAclJobViewModel(AppendAclJob job):base (job)
+        public AppendAclJobViewModel(Func<IFileShareApiAdminClient> fileShareClientFactory, AppendAclJob job, ILogger<AppendAclJobViewModel> logger) : base(job,logger)
         {
+            CloseExecutionCommand = new DelegateCommand(OnCloseExecutionCommand);
+            this.fileShareClientFactory = fileShareClientFactory;
             this.job = job;
+            this.logger = logger;          
         }
 
         public string BatchId => job.ActionParams.BatchId;
         public List<string> ReadUsers => job.ActionParams.ReadUsers;
         public List<string> ReadGroups => job.ActionParams.ReadGroups;
-        protected internal override Task OnExecuteCommand()
+
+        public DelegateCommand CloseExecutionCommand { get; }
+
+        private void OnCloseExecutionCommand()
+        {
+            ExecutionResult = string.Empty;
+            IsExecutingComplete = false;
+        }
+        protected internal override async Task OnExecuteCommand()
```

Comment:
Please tidy the indentation in the OnExecuteCommand() method

Reply:
Resolved